impress-org/givewp-next-gen
# Refactor: move form page domain into donation forms

## Description



This moves the top level FormPage domain into DonationForms.

## Affects



Viewing a donation form single page.

## Visuals



N/A

## Testing Instructions



Just make sure you can still view a donation form single using its url from the form builder.

## Pre-review Checklist
